html/src/functions.php

```diff
@@ -8,7 +8,7 @@
 function open_mysql_connection(){
     global $mysql_connection;
 
-    $host="localhost";
+    $host="mysql";
```

Comment:
I will add this to a configuration file, so the hoster can configure it himself. There is no need to add a configuration option for MariaDB User, DB, Password and Root Password too IMHO, cause by deploying using podman and docker there is no access to the database.